ts/components/FiscalCodeLandscapeOverlay.tsx

```diff
@@ -46,13 +52,26 @@ const styles = StyleSheet.create({
 export default class FiscalCodeLandscapeOverlay extends React.PureComponent<
   Props
 > {
+  private scrollViewHeight: number = 0;
+  private ScrollVewRef = React.createRef<ScrollView>();
+
   private handleBackPress = () => {
     this.props.onCancel();
     return true;
   };
 
   public componentDidMount() {
     BackHandler.addEventListener("hardwareBackPress", this.handleBackPress);
+    if (this.props.showBackSide) {
+      setTimeout(() => {
```

Comment:
using setTimeout for these operations it's not recommended. It could cause unexpected behaviours. You cannot say "in 100 milliseconds the UI is ready and updated".
Anyway you could use a life cycle method [componentDidUpdate() ](https://en.reactjs.org/docs/react-component.html#https://en.reactjs.org/docs/react-component.html#componentdidupdate)

Reply:
@Undermaken I had already tried the componentDidUpdate() method in this way:
``` typescript 
public componentDidUpdate() {
    if (this.props.showBackSide && this.ScrollVewRef.current) {
      this.ScrollVewRef.current.scrollToEnd({ animated: true });
    }
  }
```
but it seems to not work because it is not called for the initial render.

Reply:
Yes this is a solution 👍 
_onContentSizeChange_ could be also used, it's the same.

Reply:
Ok I commited the change and updated the branch from master.